main.py

```
@@ -1,44 +1,33 @@
+async def handle_message(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    user_text = update.message.text
+    headers = {
+        "Authorization": f"Bearer {OPENROUTER_KEY}",
+        "Content-Type": "application/json"
+    }
+    data = {
+        "model": MODEL_NAME,
+        "messages": [{"role": "user", "content": user_text}]
+    }
+    
+    response = requests.post("https://openrouter.ai/api/v1/chat/completions", headers=headers, json=data)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)